src/serializer/ResidualHeapVisitor.js

```diff
@@ -351,6 +400,10 @@ export class ResidualHeapVisitor {
       for (let innerName of functionInfo.unbound) {
         let residualFunctionBinding;
         let doesNotMatter = true;
+        let modified = functionInfo.modified.has(innerName);
+        // For additonal function values, we don't know if non-modified values
```

Comment:
additional

Reply:
Why don't we know that? Are we lacking a visit functions for the `Effects` that define additional functions?